src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -185,9 +249,12 @@ class TizenControls extends Controls {
     }
 
     disable(model: ViewModel): void {
+        this.api = null;
```

Comment:
Do `this.seekbar` and `this.settingsMenu` need to be cleaned up here?

Reply:
`this.seekbar` yes, but settings menu is fine. It gets destroyed in `super.disable`